README.md

```diff
@@ -80,6 +86,11 @@ the `install` command every time that you rebuild your extension. For certain in
 you might also need another flag instead of `--sys-prefix`, but we won't cover the meaning
 of those flags here.
 
+Finally, if you use a virtual environment, you need to install it as a new kernel to IPython:
```

Comment:
Do you *need* to? Like if I activate that venv and install jlab from there is this step still necessary?

Reply:
In my experience yes. No matter from which environment I launch Jupyter Lab, it always offers me all of the installed kernels, with the launching environment not necessarily being part of it.